content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -45,20 +45,20 @@ At the moment maturity varies depending on the area of ownership:
 In the last few months we’ve:
 
 - Added precise support for the JVM ecosystem (Java, Scala and Kotlin).
-- Added search-based support for Apex.
-- Shipped critical platform features necessary to set up and run auto-indexing at on-prem customers that use GCP or AWS.
+- Shipped the [executors service](https://docs.sourcegraph.com/admin/executors) that powers [auto-indexing](https://docs.sourcegraph.com/code_intelligence/explanations/auto_indexing) at on-prem customers that use GCP or AWS.
 
 Recent key learnings:
 
 - We learned that LSIF manual setup is non-trivial and requires various levels of team involvement at enterprise customers. There’s an appetite for a out-of-the-box precise code navigation solution.
 - Investing in platform performance and scalability is critical for the Global code graph vision to become a reality. Cloud is already at 2TB of data at ~40k indices. The global code graph for Java alone is somewhere between 800k to 6 million indices, so anywhere from 40TB to 300TB would be needed for the JVM code graph alone.
+- Establishing a pricing strategy for the executors service proved to be non-trivial for all of our offerings.
 
 ### Top customer, support, sales and marketing issues
```

Comment:
Is there a particular profile of customers (e.g. company size, industry, code base size, ...) that these requests come from? Or are we seeing these generally across the board? Does it make sense to try to contextualize these feedback/requests?

Reply:
We should have a better process now for getting pricing decisions (still a recent learning, but just mentioning): https://handbook.sourcegraph.com/departments/product-engineering/product/process/#executive-decisions. 

https://docs.google.com/document/d/1Hj2AYA9RAb9zo7-cC4zoIrd68QRungyeGywCf20Sfwk/edit is meant to make this decision making flow even more transparent and clear, across all of leadership (still very much WIP, but that's where the work will go).

Reply:
@yinquanteo these are patterns generally across the board. I'll add a sentence to clarify and also link to product gaps (calling out they're internal only)